src/domains/Find/page/FindIDPage.jsx

```
@@ -19,23 +22,116 @@ const FindIDPage = () => {
   } = useForm()
   const [validationStatus, setValidationStatus] = useState(null)
   const navigate = useNavigate()
+  const [isCheckingEmail, setIsCheckingEmail] = useState(false)
+  const { findID, setFindID } = useFindIDStore()
```

ESLint

[error] 'findID' is assigned a value but never used.

(no-unused-vars)